src/features/content-answers/__tests__/Answer.test.tsx

```diff
@@ -2,21 +2,32 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 
 import Answer from '../Answer';
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { ElementsXhrError } from '../../../common/types/api';
 
 describe('features/content-answers/answer', () => {
-    const renderComponent = (props: { answer: string; isLoading: boolean }) => render(<Answer {...props} />);
+    const renderComponent = (props: { answer: string; error: ElementsXhrError; isLoading: boolean }) =>
+        render(<Answer {...props} />);
 
     test('should render the answer', () => {
-        renderComponent({ answer: 'some answer', isLoading: false });
+        renderComponent({ answer: 'some answer', error: undefined, isLoading: false });
 
         const answer = screen.getByText('some answer');
         expect(answer).toBeInTheDocument();
     });
 
     test('should render the loading element', () => {
-        renderComponent({ answer: '', isLoading: true });
+        renderComponent({ answer: '', error: undefined, isLoading: true });
 
         const loadingElement = screen.getByTestId('LoadingElement');
         expect(loadingElement).toBeInTheDocument();
     });
+
+    test('should render inline error', () => {
+        renderComponent({ answer: '', error: new Error('error'), isLoading: true });
```

Comment:
I think you need to do something like a test.each to check all the possibilities and then verify the expected behavior in each of them